ddl/ddl_api.go

```diff
@@ -3955,3 +3956,128 @@ func extractCollateFromOption(def *ast.ColumnDef) []string {
 	}
 	return specifiedCollates
 }
+
+func (d *ddl) RepairTable(ctx sessionctx.Context, table *ast.TableName, createStmt *ast.CreateTableStmt) error {
+	// tableInfo and oldDBInfo's existence have been checked in preprocessor.
+	oldTableInfo, ok := (ctx.Value(domainutil.RepairedTable)).(*model.TableInfo)
+	if !ok || oldTableInfo == nil {
+		return ErrRepairTableFail.GenWithStackByArgs("Get the repaired table failed")
```

Comment:
PLease use `GenWithStack` function.

Also, how about `"Failed to get the repaired table?"`?

Reply:
Addressed.